src/ffi.rs

```diff
@@ -83,6 +83,7 @@ extern "C" {
     pub fn ada_get_search(url: *mut ada_url) -> ada_string;
     pub fn ada_get_protocol(url: *mut ada_url) -> ada_string;
     pub fn ada_get_host_type(url: *mut ada_url) -> c_uint;
+    pub fn ada_get_schema_type(url: *mut ada_url) -> c_uint;
```

Comment:
```suggestion
    pub fn ada_get_scheme_type(url: *mut ada_url) -> c_uint;
```

Reply:
Oh good spot. That's a typo so i need to send an update to Ada for this.

Reply:
Ah, saw you already fixed the typo and there's a new release already 👍 Thanks.

Reply:
Updated.